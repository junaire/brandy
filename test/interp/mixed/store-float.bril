@main() {
  one: int = const 1;
  seed: float = const 109658.0;
  rng: ptr<float> = alloc one;
  store rng seed;
  x: float = load rng;
  print x;
  free rng;
}