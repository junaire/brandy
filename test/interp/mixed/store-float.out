109658.00000000000000000
